cmd/everoute-controller/main.go

```diff
@@ -149,8 +149,9 @@ func main() {
 
 		if enableProxy {
 			if err = (&k8s.EndpointsReconcile{
-				Client: mgr.GetClient(),
-				Scheme: mgr.GetScheme(),
+				APIReader: mgr.GetAPIReader(),
```

Comment:
Add api reader to read directly from apiserver? why not read it from cache, it has some special case ？

Reply:
when list servicePort resource, the cache doesn't sync in time